locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Dockerizing a Node.js web app
+layout: docs.hbs
+---
+
+# Dockerizing a Node.js web app
+
+The goal of this example is to show you how to get a Node.js application into a
+Docker container. It assumes you have a working Docker installation and a basic
```

Comment:
> Docker installation

Could change that text to a link to here: https://docs.docker.com/engine/installation/